src/portal/interfaces/generate-portal-link-intent.interface.ts

```diff
@@ -1,3 +1,4 @@
 export enum GeneratePortalLinkIntent {
   SSO = 'sso',
+  DSYNC = 'dsync',
```

Comment:
I actually have no idea if this should be `DSync` `Dsync` or `DSYNC`! I feel like `DSync` might be right?

Reply:
Yea, enum members should be named using `PascalCase`, so either `Dsync` or `DSync, depending on what sect of naming conventions you adhered to 😅

Reply:
Thanks! I updated it to `DSync` 👍